resdk/resources/permissions.py

```diff
@@ -234,7 +234,7 @@ def _perms_to_string(self, perms):
         perms = copy.copy(perms)
 
         # Order known permissions by importance.
-        for known_perm in ['owner', 'share', 'edit', 'add', 'download', 'view']:
+        for known_perm in ['owner', 'share', 'edit', 'view']:
```

Comment:
We could abstract this bit if we made permissions a constant.

Reply:
What do you mean?
`for known_perm in self.all_permissions:` (and self.all_permissions are ordered by importance)

Reply:
For examples, define `ALL_PERMISSIONS` in `constants.py`.

Reply:
Aha, OK. Done.